cmd/info.go

```diff
@@ -81,7 +82,12 @@ func NewProjectInfo() (ProjectInfo, error) {
 			Revision: "non-git",
 		}
 	} else {
-		repo, err := repoLocation(shellOutput("git config --get remote.origin.url"))
+		cmd := exec.Command("git", "config", "--get", "remote.origin.url")
+		repoURL, err := cmd.Output()
+		if err != nil {
+			log.Println("warning: unable to get repo location info from 'origin' remote")
```

Comment:
please use the [`warn`](https://github.com/prometheus/promu/blob/master/cmd/promu.go#L135-L142) function.